test/hotspot/jtreg/vmTestbase/metaspace/stressHierarchy/common/StressHierarchyBaseClass.java

```diff
@@ -109,8 +109,13 @@ public void run() {
 
             System.out.println("Whole test took " + ((System.currentTimeMillis() - startTimeStamp)/1000/60.0) +" min");
             log.info("Test PASSED");
-        } catch (GotWrongOOMEException e) {
-            log.info("GotWrongOOMEExc: " + e.getMessage());
+        } catch (HeapOOMEException e) {
+            /**
+             * Usually this means that we got OOME:heap while trying to gain OOME:metaspace.
+             * We pass test in this case as this breaks test logic. We have dedicated test configurations
+             * for OOME:heap provoking class unloading, that why we are not missing test coverage here.
+             */
```

Comment:
This comment is unnecessary, as it adds nothing over the log message that follows.

Reply:
Thanks! I just moved it and missed that is the same as the log message.